### PR TITLE
docs(tutorial/6 - Templating Links

### PR DESCRIPTION
**Pull request by heidi-jungel**
Updating to include the alt attribute for the image.  Per WCAG 2.0 Level A 1.1.1, all images must have alternative text.  Since this is likely a tutorial developers will follow, better to include accessibility while they are learning angular and show how it can be done with angular.

### CODE REVIEW

**Comment by gilesedinburgh** [comment]
requesting to add alt attribute to tutorial file

**Comment by gilesedinburgh** [comment]
I signed it!

File: docs/content/tutorial/step_06.ngdoc [hunk 1]
```diff
@@ -43,7 +44,7 @@ __`app/index.html`:__
 ...
         <ul class="phones">
           <li ng-repeat="phone in phones | filter:query | orderBy:orderProp" class="thumbnail">
-            <a href="#/phones/{{phone.id}}" class="thumb"><img ng-src="{{phone.imageUrl}}"></a>
+            <a href="#/phones/{{phone.id}}" class="thumb"><img ng-src="{{phone.imageUrl}}" alt="{{phone.alt}}></a>
```
Comment (gkalpak) [inline docs/content/tutorial/step_06.ngdoc]: Missing closing quote.

**Comment by gkalpak** [comment]
I tool a closer look. If we are adding `alt`, I think we should be consistent. This means that similar changes are needed in steps 5, 10 and 12 as well as in the `angular-phonecat` repo.

**Comment by gilesedinburgh** [comment]
I totally agree!  We should have these in all steps for consistency. I can certainly make that request for each of these steps.

**Comment by gkalpak** [comment]
@gilesedinburgh, awesome ! I think it would be best to make all changes in one PR (plus one more for the `angular-phonecat` repo).

**Comment by gkalpak** [comment]
Closed with c2033d7.

**Pull request closed without merging**